samples/apps/cap/py/src/CAPActor.py

```diff
@@ -0,0 +1,78 @@
+from DebugLog import Debug, Info
+import zmq
+import threading
+import traceback
+from CAPConstants import Termination_Topic, xpub_url
+
+
+class CAPActor:
```

Comment:
Given CAP is the the package I think prefixing things with `CAP` is not necessary and adds noise

Reply:
Or.... hear me out... we prefix everything with "NS"  :)